core/build.gradle

```diff
@@ -12,31 +12,17 @@ tasks.withType(JavaCompile) {
     options.compilerArgs << "-source" << "1.6" << "-target" << "1.6"
 }
 
-// Build up a KeenVersion.java akin to the Android BuildConfig.java, but available in all projects.
-// We set Implementation-Version on the Manifest, but reading Package.getImplementationVersion()
-// on the core Package seems return null in the Android client, so instead generate this. Also, the
-// Manifest info isn't set for unit tests.
-task generateVersionFile {
-    def outputDir = file("$buildDir/generated/source")
-    outputs.dir outputDir
+task verifyVersionFile {
```

Comment:
yeap, it didn't work as I hadn't committed KeenVersion.java

let me clarify why I changed that: by now KeenVersion has been generated on the fly. It was fine from the perspective of using gradle, but development with Intellij was tricky - I had to create this file in sources manually and exclude it from git - but then gradle build failed because of duplicated class.. I've changed that I little bit - added KeenVersion to files tree and replaced generateVersionFile with verifyVersionFile that checks whether versions are consistent.

pros: everything works out of the box (both for gradle and intellij)
cons: there is necessity to bump version in 3 places instead of 2 (readme, gradle pros, KeenVersion)

wdyt about it?